doc/source/reference/arrays.scalars.rst

```diff
@@ -65,19 +65,20 @@ Some of the scalar types are essentially equivalent to fundamental
 Python types and therefore inherit from them as well as from the
 generic array scalar type:
 
-====================  ================================
-Array scalar type     Related Python type
-====================  ================================
-:class:`int_`         :class:`IntType` (Python 2 only)
-:class:`float_`       :class:`FloatType`
-:class:`complex_`     :class:`ComplexType`
-:class:`bytes_`       :class:`BytesType`
-:class:`unicode_`     :class:`UnicodeType`
-====================  ================================
+====================  ===================  =============
+Array scalar type     Related Python type  Inherits?
+====================  ===================  =============
```

Comment:
`np.datetime64` and `np.timedelta64` are still missing.
`np.void` _might_ also be worth mentioning, even though it's strictly speaking not a scalar.

Reply:
This table is not exhaustive, it only covers things with related builtin types. Void doesn't have a corresponding builtin type, but I suppose `np.datetime64` and `np.timedelta64` do: added.